File: problems/python/1.py
#!/usr/bin/python

# Project Euler, Problem #1
#
# Find the sum of all the multiples of 3 or 5 below 1000.

def isMultof3or5(number): 
    if ((number % 3 == 0) | (number % 5 == 0)):
        return True
    else: 
        return False 

multlist = []
for i in range(3,1000):
    if (isMultof3or5(i)):
        multlist.append(i)

sum = 0
i=0
for i in range(len(multlist)):
   sum += multlist[i]

print(sum) 

File: problems/c/1.c
#include <stdio.h> 

/* Program to solve problem 1 of the Euler Project. 
 *
 * Find the sum of all the multiples of 3 or 5 below 1000. */

int divisible(int n) {
    return ((n%3==0)||(n%5==0));
} 

int main() {
    int sum = 0;
    int i; 
    for (i = 0; i < 1000; i++) {
       if (divisible(i)) 
           sum += i; 
    }
    printf("Sum = %d\n" ,sum); 
    return 0; 

}

